examples/backbone.localStorage.js

```diff
@@ -114,54 +114,43 @@ _.extend(Backbone.LocalStorage.prototype, {
 Backbone.LocalStorage.sync = window.Store.sync = Backbone.localSync = function(method, model, options) {
   var store = model.localStorage || model.collection.localStorage;
 
-  var resp, errorMessage, syncDfd = $.Deferred && $.Deferred(); //If $ is having Deferred - use it.
-
-  try {
-
-    switch (method) {
-      case "read":
-        resp = model.id != undefined ? store.find(model) : store.findAll();
-        break;
-      case "create":
-        resp = store.create(model);
-        break;
-      case "update":
-        resp = store.update(model);
-        break;
-      case "delete":
-        resp = store.destroy(model);
-        break;
+  var resp, errorMessage;
+
+  return new Promise(function(resolve, reject) {
```

Comment:
Did this changeset break `backbone.localStorage`? For this to be accepted I don't think we should be breaking the major popular plugins in the ecosytem.